tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,14 +1,14 @@
-# How to make smartfs user image for TizenRT
+# How to make jSmartFs user image for TizenRT
```

Comment:
missed changes in 
tools/nxfuse/src/main.c
tools/nxfuse/src/tinyara_services.c